rules/touch.go

```diff
@@ -17,7 +19,25 @@ func isMatchTouch(cmd models.Command) bool {
 }
 
 func getUndoTouch(cmd models.Command) string {
-	var filePaths = parse.GetFilePathsFromTokens(cmd.Tokens)
+	var touchFlags struct {
+		AccessAndModification string `short:"A"`
+		Access                bool   `short:"a"`
+		DontCreate            bool   `short:"c"`
+		Force                 bool   `short:"f"`
+		ChangeSymLink         bool   `short:"h"`
+		Modification          bool   `short:"m"`
+		SpecFile              string `short:"r"`
+		Time                  string `short:"t"`
+	}
+	filteredTokens, err := flags.ParseArgs(&touchFlags, cmd.Tokens[1:])
```

Comment:
I think I was wrong about having a field in Command model for it.. Don't see any use cases where you would rather want to set it on the model instead of just having it in a variable which you can just use. What do you think?